pheps/phep-0001.md

```diff
@@ -0,0 +1,366 @@
+Created: 20-Oct-2023
+Post-History: 24-Oct-2023, 31-Oct-2023, 9-Nov-2023, 27-Nov-2023, 06-Dec-2023
+```
+
+# What is a PHEP?
+<a name="what-is-a-phep"></a>
+PHEP stands for PyHC Enhancement Proposal. A PHEP provides information to the Python in Heliophysics Community (PyHC), describes its processes, or establishes standards for its packages.
+
+PHEPs are the primary mechanism for proposing standards, collecting community input, and documenting the design decisions underlying PyHC standards and recommendations. The PHEP process develops and documents consensus within the community. The PHEP author is responsible for building this consensus and documenting dissenting opinions.
+
+Because PHEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal. This record is available by the normal git commands for retrieving older revisions and can be browsed [on GitHub](https://github.com/heliophysicsPy/standards).
+
+# PHEP Motivation
+<a name="phep-motivation"></a>
+The [PyHC standards](https://doi.org/10.5281/zenodo.2529131) have not been updated since their initial approval in 2018, and there is no established process for updating them. PyHC discussions often involve what packages "should" do, or "should be required" to do. For standards to be effective, they need to be seen as legitimate by those they may be imposed on.
+
+A mechanism is needed to capture the sense of the community and allow it to progress while recognizing that the community is broad, diffuse, and diverse, and not all stakeholders are present for all conversations.
+
+# PHEP Audience
+<a name="phep-audience"></a>
+The typical audience for PHEPs is the developers and user communities of PyHC packages. Other parts of the PyHC community (hereafter "community") may also use the process (particularly for Informational PHEPs) to document conventions and to manage complex design coordination problems that require collaboration across multiple projects.
+
+This PHEP refers to "PyHC leadership": individuals or groups with statutory authority over the PyHC project. This may be the Principal Investigator or other individuals or groups appropriately designated in the future, such as a steering committee.
+
+# PHEP Types
+<a name="phep-type"></a>
+There are three types of PHEP:
+
+1. A **Standards** Track PHEP describes a new or modified standard for PyHC packages. These standards are requirements that PyHC packages are expected to follow, similar to the original [PyHC standards](https://doi.org/10.5281/zenodo.2529131). Examples include testing requirements, OS support, and dependency handling.
+
+2. An **Informational** PHEP describes a design issue, or provides general guidelines or information to the community, but does not propose a standard. Informational PHEPs do not necessarily represent a community recommendation, so users and implementers are free to ignore Informational PHEPs or follow their advice.
```

Comment:
If these are not heavily suggested to be followed, do they need to go through the stringent process outlined here? My thinking is to keep all PHEP types the same, but I figured I'd raise the question.